editoast/src/modelsv2/light_rolling_stock.rs

```diff
@@ -65,97 +59,34 @@ pub struct LightRollingStockModel {
 }
 
 impl LightRollingStockModel {
-    pub async fn with_liveries(
-        self,
+    pub async fn liveries(
+        &self,
         conn: &mut DbConnection,
-    ) -> Result<LightRollingStockWithLiveriesModel> {
+    ) -> Result<Vec<RollingStockLiveryMetadata>> {
```

Comment:
```suggestion
    ) -> Result<Vec<RollingStockLiveryModel>> {
```

1. We have two very similar structs for liveries in `modelsv2`: `RollingStockLiveryModel` and `RollingStockLiveryMetadataModel`. I suggest getting rid of the latter.
2. We're in the `modelsv2` module, implementing an operation on a model that fetches another model. There's no need to convert into a schema this early.